events/models.py

```diff
@@ -3,31 +3,48 @@
 
 
 class Event(models.Model):
-    event = models.CharField(max_length=100, blank=True)
-    description = models.CharField(max_length=1000, blank=True)
-    start_time = models.DateTimeField()
-    end_time = models.DateTimeField()
+    priority = models.IntegerField(blank=True)
+    event = models.CharField(max_length=100, blank=False)
+    description = models.CharField(max_length=2000, blank=True)
     prize = models.CharField(max_length=20, blank=True)
-    venue = models.CharField(max_length=10, blank=True)
-    image = models.ImageField(upload_to='event-images/', null=True, blank=True)
+    team_size = models.CharField(max_length=20, blank=True)
```

Comment:
Shouldn't it be `IntegerField`?

Reply:
team size is like 1 - 3 variable in some events, that's why I used Charfield

Reply:
Check `IntegerRange` field

Reply:
Sir, that's specific to PostgreSQL and in this project, we are using SQLite